src/mutations/appData.ts

```diff
@@ -75,10 +87,46 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
       }
```

Comment:
Is the double insertion check not required in this mutation? From my understanding, the data structure is also a list here.

Reply:
As we patch the data here, we don't necessarily need to care about double patching. There is no insertion, you just overwrite the data.